test/integration/all.py

```diff
@@ -1,28 +1,32 @@
 import os
 import sys
 from aibolit.config import Config
+import tqdm
 
 # TODO: fix all errors in the patterns/metrics and make these lists empty
 EXCLUDE_PATTERNS = ['P31', 'P32']
 EXCLUDE_METRICS = []
 
 current_path: str = os.path.dirname(os.path.realpath(__file__))
-for filename in os.listdir(current_path + '/samples'):
+print('Processed files in testing:')
+
+for filename in tqdm.tqdm(os.listdir(current_path + '/samples')):
 
     for pattern in Config.get_patterns_config()['patterns']:
         if pattern['code'] in EXCLUDE_PATTERNS:
             continue
         try:
             path_to_file = os.path.join(current_path, 'samples', filename)
             pattern['make']().value(path_to_file)
-        except Exception:
+        except Exception as e:
             print(
                 "Error apply the pattern:",
                 pattern['name'],
                 pattern['code'],
                 "to file",
                 filename
             )
+            print(e)
```

Comment:
We better make a singe `print` statement with one formatting, maybe multi line string. So it could be read as a single sentence.
Same in another `except`

Reply:
done